WordPress/src/main/java/org/wordpress/android/util/WPLegacyMigrationUtils.java

```diff
@@ -144,10 +145,10 @@ public static boolean hasSelfHostedSiteToMigrate(Context context) {
         }
     }
 
-    private static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
+    static List<SiteModel> getSelfHostedSitesFromDeprecatedDB(Context context) {
```

Comment:
I've opened a new issue to fix `SiteModel` fields not populated during migration (including `isVisible`): https://github.com/wordpress-mobile/WordPress-Android/issues/5306.